example-playbooks/replace_node/replace_node.yml

```diff
@@ -24,6 +24,30 @@
       when: hostvars[new_node]['broadcast_address'] in scylla_seeds or new_node in scylla_seeds
       run_once: true
 
+    - name: Stop scylla-server on the node being replaced
+      block:
+        - name: run nodetool drain
+          shell: |
+            nodetool drain
+
+        - name: wait for DRAIN to complete
+          shell: |
+            journalctl -u scylla-server --since "1 minute ago" | grep 'storage_service' | grep 'DRAINED'
+          retries: 3000
+          delay: 1
+          register: drain_result
+          until: drain_result.rc == 0
+          ignore_errors: true
+
+        - name: Stop Scylla service
```

Comment:
Continuing my previous point: you want to time limit this operation by, say 60 seconds and after that `kill -9` scylla. We really don't care about stop scylla "nice" on this node.

Reply:
Done